### DISCUSSION TITLE
student devloper pack

### DISCUSSION THREAD

**Discussion by sujalsrichandan2003** [community/community#68248] [GitHub Education]
### Select Topic Area

Question

### Body

i am student of btech 2nd year , i have applied 6 time every time i get rejected. in my college id card in front side my details and college name written and back side my graduation year (2026) but git hub rejected that and when i give my enrollment letter what i got from my college that also not accepted. what should i do now?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @sujalsrichandan2003 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
